packages/tools/pie-wrapper-react/__tests__/__snapshots__/wrapper.test.js.snap

```diff
@@ -3,48 +3,50 @@
 exports[`React Wrapper should be added from mock custom elements JSON 1`] = `
 "
 import * as React from 'react';
+import { createComponent, EventName } from '@lit-labs/react';
 import { MockComponent as MockComponentReact } from './index';
-import { createComponent } from '@lit-labs/react';
-import { EventName } from '@lit-labs/react';
 
 export const MockComponent = createComponent({
     displayName: 'MockComponent',
     elementClass: MockComponentReact,
     react: React,
     tagName: 'mock-component',
-    events: { onCustomEvent: 'CustomEvent' as EventName<CustomEvent>, onAnotherCustomEvent: 'AnotherCustomEvent' as EventName<CustomEvent>, },
+    events: {
+        onCustomEvent: 'CustomEvent' as EventName<CustomEvent>, 
```

Comment:
I would suggest as a rule of thumb we keep our events to single words if possible. I think `pie-modal-open` for example could simply be `open`. Which would become `onopen`. Keeping them as short as possible will improve readability.

Reply:
Bear in mind that native browser events follow lowercase, no hyphen formats. Such as `onreadystatechange`, `touchstart`, `fullscreenerror`. Depends how much we want to mirror native naming conventions

Reply:
@raoufswe what we could always do is keep events in the web components like `modal-open` but camel case them for the react wrapper so it would become `onModalOpen`. Perhaps that would be a best of both solution?

Reply:
Agree with this – happy for this to be done in a follow up PR

Reply:
Ta - will raise this shortly